fastai/transforms.py

```diff
@@ -465,6 +478,21 @@ def __call__(self, im, y=None): return compose(im, y, self.tfms)
 
 def image_gen(normalizer, denorm, sz, tfms=None, max_zoom=None, pad=0, crop_type=None,
               tfm_y=None, sz_y=None, pad_mode=cv2.BORDER_REFLECT):
+    """
+    Returns transformer for specified image operations.
```

Comment:
Rather than writing 'Returns' at the start of the docstring, instead say something like 'generate a standard set of transformations' for the summary, and then create a 'returns' section in the docstring with the details of the return value. For some suggestions, see the [numpy docstring page](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).

Reply:
Love this. Brings order to the universe. 
Now I know I've been doing everything wrong, do I have your blessing to change your previous doc strings too as well as mine? 

Thanks!